src/Reporters/MailBrokenLinks.php

```diff
@@ -71,6 +71,8 @@ public function crawlFailed(
         RequestException $requestException,
         ?UriInterface $foundOnUrl = null
     ) {
-        return;
+        $url->foundOnUrl = $foundOnUrl;
```

Comment:
Is this necessary? I don't think `https://laravel.com/api/5.6/Illuminate/Auth/Events/Login.html` has a `foundOnUrl` property.

Reply:
I think it is necessary.
The `$url` incoming here is a `\GuzzleHttp\Psr7\Uri`which doesnt have a `foundUrl` property.